proto/api/v2/compliance_profile_service.proto

```diff
@@ -32,16 +32,19 @@ message ComplianceProfileSummary {
   string title = 4;
   int32 rule_count = 5;
   string profile_version = 6;
+  string standard = 7;
```

Comment:
placeholder

Reply:
I believe we should refrain from including the `standard` field in our API responses prematurely as it locks us in without providing significant value. Which value does it provide if it is added?

Reply:
yeah I think I will take it out.  Basically it would provide the benchmark, but I think it would be best to add that when we are going to use it.